src/Skia/Avalonia.Skia/Compatibility/SkiaCompat.SKPath.cs

```diff
@@ -0,0 +1,17 @@
+using System;
+using System.Runtime.InteropServices;
+using SkiaSharp;
+
+namespace Avalonia.Skia;
+
+internal static unsafe partial class SkiaCompat
+{
+    public static void CTransform(this SKPath path, ref SKMatrix matrix)
+    {
+        fixed (SKMatrix* m = &matrix)
+            sk_path_transform(path.Handle, m);
```

Comment:
This native method is the same for both new and old SkiaSharp APIs. Difference is only in managed method `SKPath.Transform(SKMatrix)` which was changed to `SKPath.Transform(in SKMatrix)`